### DISCUSSION TITLE
新人报道，该做第一件事了！

### DISCUSSION THREAD

**Discussion by alvwhddk** [share121/inter-knot#5276] [灌水]
新人进入的第一件事当然就是找使用指南和新人必知了(╹ڡ╹ )(╹ڡ╹ )
不知道我这么发出去对不对，话说帖子壁纸怎么更换，我连名字都不晓得去哪更改，pc端虽然登录了，但是好像一直显示未登录，很神奇
爱来自新艾利都⎛⎝≥⏝⏝≤⎛⎝⎛⎝≥⏝⏝≤⎛⎝

**Comment by KELERSAMA** [comment]
可以看看
#4990   搜索词：绳网常见问题的解决方法 
#4628   搜索词：绳网社区规则详情与准则 
#1436   搜索词：如何让你的帖子更加与众不同 
#4639   搜索词：在github使用米游社表情包
#2106     顺带辟谣一下
<img src="https://img-static.mihoyo.com/communityweb/upload/408018316100a8a8a846c68acedaa191.gif" alt="米游姬 卖萌" width="75" >

**Reply by alvwhddk** [reply]
好好好，原来有这么多可以看(/≧▽≦)/(/≧▽≦)/
